### DISCUSSION TITLE
Can not get the demo code connect to wallet extension

### DISCUSSION THREAD

**Discussion by robinboening** [cardano-foundation/cardano-connect-with-wallet#80] [Q&A]
Hey,

I am having an issue with setting up a connection to a wallet using the [demo code](https://github.com/cardano-foundation/cardano-connect-with-wallet/tree/main/html/cardano-connect.js)

I am using esbuild in my application and import `CardanoConnectWithWalletCore` from '@cardano-foundation/cardano-connect-with-wallet-core' (v0.2.2), copied the code from `cardano-connect.js` and added a `div#cardano-connect-container` in my html.

The dropdown renders fine and when I click on one of the available wallet items I can confirm `this.wallet.connectToWallet` gets executed and the Promise gets resolved. The issue is after this nothing visible happens, no error, and it gets hard to debug for me.

I would expect it to trigger the wallet to request for permission, or just connect.

Or is it just a user error and I am missing something obvious?

Thank you!
Robin

**Comment by fabianbormann** [comment]
Hey @robinboening, sorry I was not pinged about a new discussion thread here 🙈 .. did you make sure to have a wallet account created in your wallet app and did you check the network? Sorry again for the delay ..
